website/docs/getting-started.md

```diff
@@ -0,0 +1,78 @@
+---
+id: getting-started
+title: Getting Started
+slug: /
+---
+
+# AI CLI
+
+AI assistant in your terminal.
+
+```
+$ ai "Tell me an interesting fact about the Roman Empire"
+```
+
+## Installation
+
+```sh
+npm install -g @callstack/ai-cli
+```
+
+## Setup
+
+### API key
+
+You will need a valid API key from one of supported AI inference providers: OpenAI or Perplexity. Using these APIs is paid but the costs should be very low, typically much less than $0.01 per interaction.
+
+### Quick config
+
+Run `ai init` to create a basic `~/.airc.json` config file with your OpenAI or Perplexity API keys.
+
+Read more about the [configuration file](config-file) format and available option.
+
+## Usage
+
+You can invoke `ai` either in single-answer mode:
+
+```
+$ ai "Tell me a joke"
+me:  Tell me a joke
+ai:  Why don't scientists trust atoms? Because they make up everything!
+```
+
+Or you can start a longer conversation using interactive mode (`-i` or `--interactive` option):
```

Comment:
My issue with that is: interactive mode is by default. So this "single-answer" will not really but a "single-answer" but an initial prompt. We should be more explicit with that